addons/contour/1.10.1/contour.yaml

```diff
@@ -1351,7 +1351,7 @@ rules:
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: contour-certgen-v1.10.1
+  name: contour-certgen-v1.10.1-1
```

Comment:
Changed these to prevent immutability errors with the new image name when running this against an existing cluster.